rust/parquet/src/arrow/arrow_writer.rs

```diff
@@ -53,13 +56,48 @@ impl<W: 'static + ParquetWriter> ArrowWriter<W> {
     pub fn try_new(
         writer: W,
         arrow_schema: SchemaRef,
-        props: Option<Rc<WriterProperties>>,
+        props: Option<WriterProperties>,
```

Comment:
@maxburke I've had to change this to take the `WriterProperties` directly so I can mutate it to add the Arrow serialized schema. This affects your branch.

Reply:
:+1: